packages/app-store/googlecalendar/lib/GoogleMeetService.ts

```diff
@@ -0,0 +1,116 @@
+import { JWT } from "googleapis-common";
+
+import { GoogleAdminDirectoryService } from "./GoogleAdminDirectoryService";
+
+export interface MeetSpace {
+  name: string;
+  meetingCode: string;
+  meetingUri: string;
+}
+
+export interface MeetParticipant {
+  name: string;
+  email?: string;
+  signedinUser?: {
+    user: string;
+  };
+}
+
+export interface MeetConferenceRecord {
+  name: string;
+  startTime: string;
+  endTime: string;
+  space: string;
+  expireTime: string;
+}
+
+export class GoogleMeetService {
```

Comment:
This service is responsible for making API requests to https://developers.google.com/workspace/meet/api/guides/overview